Util.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle;
+
+class Util
+{
+    /**
+     * @param string $filename
+     *
+     * @return array
+     */
+    public static function extractCatalogueInformationFromFilename(string $filename)
+    {
+        [$domain, $locale, $extension] = explode('.', basename($filename), 3);
+
+        // Domains using ICU Message Format suffix their names with "+intl-icu".
+        // See https://symfony.com/doc/current/translation/message_format.html#using-the-icu-message-format
+        if (false !== $icuPos = strpos($domain, '+intl-icu')) {
+            $domain = substr($domain, 0, $icuPos);
```

Comment:
This looks wrong to me. If you remove the `+intl-icu` suffix, you loose the info that these messages use the ICU format. The JS code will need this info.

Reply:
That's why I thought some minutes ago before your early review.

Maybe return a 4th value `$format`